docs/extras/xdebug.md

```diff
@@ -14,6 +14,11 @@ To use XDebug with PHPStorm, change the `php_xdebug_idekey` variable as shown be
 ```yaml
 php_xdebug_idekey: PHPSTORM
 ```
+In order to pass the Web Server Debug Validation, you'll need to also add the following your `config.yml`
+
+```yaml
+php_xdebug_remote_host: "{{ vagrant_hostname }}"
```

Comment:
@cdmo if you're using PHPStorm I don't think you need to set it as the IP is automatically resolved with the network connection of the browser running on the host.

However, I've found you do need to set it when using xdebug in CLI.